src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

```diff
@@ -116,12 +116,20 @@ kibana_vars=(
     server.compression.referrerWhitelist
     server.cors
     server.cors.origin
-    server.customResponseHeaders
+    server.securityResponseHeaders.strictTransportSecurity
+    server.securityResponseHeaders.strictTransportSecurity.maxAge
+    server.securityResponseHeaders.strictTransportSecurity.includeSubDomains
+    server.securityResponseHeaders.strictTransportSecurity.preload
+    server.securityResponseHeaders.xContentTypeOptions
+    server.securityResponseHeaders.referrerPolicy
+    server.securityResponseHeaders.permissionsPolicy
+    server.securityResponseHeaders.disableEmbedding
     server.customResponseHeaders
     server.defaultRoute
     server.host
     server.keepAliveTimeout
     server.maxPayloadBytes
+    server.maxPayload
```

Comment:
I added `server.maxPayload` but left the deprecated `server.maxPayloadBytes` for now (see 8cc60bf)